### PR TITLE
[RUN-5071] Check for moveIndependently option in hijackMovesForGroupe…

### PR DESCRIPTION
**Pull request by MichaelMCoates**
Associated with this `js-adapter` PR: https://github.com/HadoukenIO/js-adapter/pull/278
JIRA Ticket: https://appoji.jira.com/browse/RUN-5071

**[RUN-5071] Check for moveIndependently option in hijackMovesForGroupedWindows. Adjust how the payload is handled in setWindowBounds to account for new options**

**Description**
When we hijack the move/resize functions for grouped windows, we now check for the presence of a `moveIndependently` option in the options object. If that option is set to `true`, we move that window independently of the rest of its group.

**Questions:** 
Automated tests for all functions have been added to the `testing-dashboard`, but are currently private. When should I make them public? 

Should I also add in the relevant code examples/documentation for the corresponding `js-adapter` PR, or should we not publicize these options yet?

#### Checklist

- [X] PR description included and stakeholders cc'd
- [X] `npm test` passes
- [X] automated tests are [changed or added](https://testing-dashboard.openfin.co/#/app/dashboard)
- [ ] manual tests are [changed or added](https://github.com/openfin/test_apps)
- [?] relevant documentation is [changed or added](https://github.com/hadoukenio/js-adapter)
- [X] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [X] PR release notes describe the change in a way relevant to app-developers

#### Release Notes

- Window movement and resize APIs can now be used to move and/or resize a window without moving or resizing the rest of the group.

Notes:

### CODE REVIEW

File: src/browser/api_protocol/api_handlers/window.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ function stopFlashWindow(identity: Identity, message: APIMessage, ack: Acker): v
 
 function setWindowBounds(identity: Identity, message: APIMessage, ack: Acker): void {
     const { payload } = message;
-    const { top, left, width, height } = payload;
+    const { top, left, width, height } = payload.bounds;
```
Comment (pbaize) [inline src/browser/api_protocol/api_handlers/window.ts]: see comment in js adapter, leave these top level in the payload to ensure backwards compatibilty

Comment (MichaelMCoates) [reply src/browser/api_protocol/api_handlers/window.ts]: Ahhh duh, reverted.

File: src/browser/api_protocol/api_handlers/grouped_window_moves.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ export function hijackMovesForGroupedWindows(actions: ActionSpecMap) {
                     const { payload } = message;
                     const { uuid, name } = getTargetWindowIdentity(payload);
                     const wrapped = getWindowByUuidName(uuid, name);
-                    if (wrapped && wrapped.groupUuid) {
+                    if (wrapped && wrapped.groupUuid && !payload.options.moveIndependently) {
```
Comment (pbaize) [inline src/browser/api_protocol/api_handlers/grouped_window_moves.ts]: Check if options is undefined for old runtimes  adapters

Comment (MichaelMCoates) [reply src/browser/api_protocol/api_handlers/grouped_window_moves.ts]: Done!

Comment (pbaize) [reply src/browser/api_protocol/api_handlers/grouped_window_moves.ts]: :neckbeard: assign that to a variable rather than modifying payload

**Pull request merged**